pyscriptjs/src/components/pytitle.ts

```diff
@@ -1,4 +1,4 @@
-import { addClasses, htmlDecode, ensureUniqueId } from '../utils';
```

Comment:
@antocuni  just checking, should pytitle call `ensureUniqueId` as well?

Reply:
I think that `py-title` should die in flames 😅.
Looking at the code I think it should because it's indeed using `this.id`.
But on the other hand, I don't think that the id is used for anything useful, so we could just avoid using it.

Reply:
Haha removing things is always fun!

Awesome I just wanted to be sure that I didn't removed something that we would use in the future 😀